src/table/body/table-body.component.ts

```diff
@@ -41,14 +41,26 @@ import { TableRowSize } from "../table.types";
 						'tbody_row--error': !model.isRowSelected(i) && model.getRowContext(i) === 'error'
 					}">
 				</tr>
-				<tr
-					*ngIf="model.isRowExpanded(i) && !model.isRowFiltered(i)"
-					ibmTableExpandedRow
-					ibmExpandedRowHover
-					[row]="row"
-					[expanded]="model.isRowExpanded(i)"
-					[skeleton]="skeleton">
-				</tr>
+				<ng-container *ngIf="model.isRowExpanded(i) && !model.isRowFiltered(i)">
```

Comment:
Kinda unrelated to this PR, but since you're digging around in here anyway - If you check out the react storybook you'll see that the child expandable rows are always present in the DOM. They use CSS to toggle the visibility. The big advantage there is that they already have code to handle striping the rows correctly - right now our rows end up with some weird styles when striped.

It would also be one less DOM change, which is always good from a performance perspective.

Reply:
I looked into it briefly and it's not clear how css way will work for the rows. We already have an issue open for the striping, so let's do this as part of that issue.

Reply:
Sounds like a plan 👍